README.md

```diff
@@ -43,3 +44,24 @@ If your Kafka is secured you will need to enable the security configuration opti
 To increase the logging level to debug provide a system property at start time: `-Dlog.level=debug`.
 
 To increase the Kafka client logging level to info provide a system property at start time: `-Dlog.level.kafka=info`.
+
+## Build prerequisites
+
+To build this project you will need:
+
+- maven `mvn`
```

Comment:
Capital M for Maven. And we should link to places to get the prereqs.

Reply:
Can do!